test/specs/runLwcTests.e2e.ts

```diff
@@ -0,0 +1,416 @@
+      expect(terminalText).toContain('Tests:       4 passed, 4 total');
+      expect(terminalText).toContain('Snapshots:   0 total');
+      expect(terminalText).toContain('Ran all test suites.');
+    });
+
+    step('SFDX: Refresh Lightning Web Component Test Explorer', async () => {
+      utilities.log(
+        `${testSetup.testSuiteSuffixName} - SFDX: Refresh Lightning Web Component Test Explorer`
+      );
+      const workbench = await (await browser.getWorkbench()).wait();
+      await utilities.runCommandFromCommandPrompt(workbench, 'Testing: Focus on LWC Tests View', 1);
+      // Run command SFDX: Refresh Lightning Web Component Test Explorer
+      await utilities.runCommandFromCommandPrompt(
+        workbench,
+        'SFDX: Refresh Lightning Web Component Test Explorer',
+        2
+      );
+      // Open the Tests Sidebar
+      const sidebar = workbench.getSideBar();
+      const sidebarView = sidebar.getContent();
+      const lwcTestsSection = await sidebarView.getSection('LWC TESTS');
+      expect(lwcTestsSection.elem).toBePresent();
+      let lwcTestsItems = (await lwcTestsSection.getVisibleItems()) as TreeItem[];
+
+      // Run command SFDX: Run All Lightning Web Component Tests
+      await utilities.runCommandFromCommandPrompt(
+        workbench,
+        'SFDX: Run All Lightning Web Component Tests',
+        2
+      );
+
+      // Get tree items again
+      lwcTestsItems = (await lwcTestsSection.getVisibleItems()) as TreeItem[];
+
+      // Verify the tests that ran are labeled with a green dot on the Test sidebar
+      for (const item of lwcTestsItems) {
+        const icon = await (await item.elem).$('.custom-view-tree-node-item-icon');
+        const iconStyle = await icon.getAttribute('style');
+        // Try/catch used to get around arbitrary flaky failure on Ubuntu in remote
+        try {
+          expect(iconStyle).toContain('testPass');
+        } catch {
+          utilities.log('ERROR: icon did not turn green after test successfully ran');
+        }
+      }
+
+      // Run command SFDX: Refresh Lightning Web Component Test Explorer again to reset status
+      await utilities.runCommandFromCommandPrompt(
+        workbench,
+        'SFDX: Refresh Lightning Web Component Test Explorer',
+        2
+      );
+
+      // Get tree items again
+      lwcTestsItems = (await lwcTestsSection.getVisibleItems()) as TreeItem[];
+
+      // Verify the tests are now labeled with a blue dot on the Test sidebar
+      for (const item of lwcTestsItems) {
+        const icon = await (await item.elem).$('.custom-view-tree-node-item-icon');
+        const iconStyle = await icon.getAttribute('style');
+        // Try/catch used to get around arbitrary flaky failure on Ubuntu in remote
+        try {
+          expect(iconStyle).toContain('testNotRun');
+        } catch {
+          utilities.log('ERROR: icon did not turn green after test successfully ran');
```

Comment:
```suggestion
          utilities.log('ERROR: icon is not blue');
```

Reply:
I actually prefer the original error message, it's also the same we use for apex tests and is more descriptive